Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM tiangolo/uvicorn-gunicorn-fastapi:python3.8-slim as compile-image
+
+WORKDIR /app
+RUN apt-get update && \
+    apt-get -y install gcc mono-mcs && \
+    rm -rf /var/lib/apt/lists/*
+COPY ./requirements.txt /app/requirements.txt
+RUN pip install --no-cache-dir --user -r /app/requirements.txt; rm -r /tmp 
+
+FROM tiangolo/uvicorn-gunicorn-fastapi:python3.8-slim
+COPY --from=compile-image /root/.local /root/.local
+COPY ./app /app/app
+COPY ./crawler /app/crawler
+COPY ./scrapy.cfg /app/scrapy.cfg
+ENV PYTHONPATH=/app
```

Comment:
Not sure what's going on here with the pythonpath. Just changing `WORKDIR` to `app` should add the necessary modules to pythonpath

Reply:
Right